source/slang/slang-capabilities.capdef

```diff
@@ -387,7 +385,7 @@ alias bufferreference_int64 = bufferreference + GL_EXT_shader_explicit_arithmeti
 
 
 alias sm_4_0 = _sm_4_0
-             | glsl_spirv_1_0 + _GL_ARB_sparse_texture_clamp + _GL_EXT_samplerless_texture_functions
+             | glsl_spirv_1_0 + _GLSL_150 + _GL_ARB_sparse_texture_clamp + _GL_EXT_samplerless_texture_functions
```

Comment:
Wouldn't this make this change "breaking change"?
If the customers were using sm_4_0 functions with GLSL version lower than GLSL 1.5, it may cause a problem, isn't it?

Reply:
This PR is a breaking change if someone previously used capabilities (there were quite a few pre-existing issues that allowed incorrect capabilities to work)

Reply:
Fixing a bug is one thing and breaking the content pipeline is another things.
If the application side will be impacted by "correct" behavior of Slang, we need to mention it on our release note.

Reply:
If we are doing this right, then there shouldn't be a breaking change. The reason we are doing the capability system is to make it a type error if the user uses an intrinsic that is not available on a platform. Without the capability system check, that will just be some invalid code comming out from slang that won't compile by the down stream compiler.

With capability system we will instead diagnose an error. So although we are seemly reporting more errors in code that wasn't erroring before, we are just front loading the error from downstream to slang. For actually valid code we shouldn't suddenly be reporting errors with the capability system in place, if we do it right.

Reply:
I suggest we bring up the `ignore-capabilities` option in release notes.